File: backend/src/main/java/com/example/backend/model/Tipocontribuyente.java
package com.example.backend.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import lombok.Data;

@Data
@Entity
public class Tipocontribuyente {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Integer id_tipo_contribuyente;

	@Column(name = "nombre", nullable = true, unique = true)
	private String nombre;

	@Column(name = "estado", nullable = true)
	private Boolean estado;

}

File: frontend/src/app/models/tipocontribuyente.ts
export class Tipocontribuyente {
  id_tipo_contribuyente?: number;
  nombre: string;
  estado: boolean;

  constructor(id_tipo_contribuyente: number, nombre: string) {
      this.nombre = nombre;
      this.id_tipo_contribuyente = id_tipo_contribuyente;
  }
}